src/index.css

```diff
@@ -366,6 +366,7 @@ input[type="file"] {
 
 .btn {
   border: 0;
+  width:100%;
```

Comment:
I don't like these types of edits - you are editing global css for a button, just to change a couple of buttons on your PR. Can you explain the reasoning for this? Have you gone through and checked every single button throughout the website, to check if it's still okay and not warped/ugly?